tests/tracking/fluent/test_fluent.py

```diff
@@ -1690,6 +1690,35 @@ def test_runs_are_ended_by_run_id():
     assert mlflow.active_run() is None
 
 
+def test_create_logged_model_tags_from_context():
+    expected_tags = {
+        mlflow_tags.MLFLOW_SOURCE_NAME: "source_name",
+        mlflow_tags.MLFLOW_SOURCE_TYPE: SourceType.to_string(SourceType.NOTEBOOK),
+        mlflow_tags.MLFLOW_GIT_COMMIT: "1234",
+    }
+
+    source_name_patch = mock.patch(
+        "mlflow.tracking.context.default_context._get_source_name",
+        return_value=expected_tags[mlflow_tags.MLFLOW_SOURCE_NAME],
+    )
+    source_type_patch = mock.patch(
+        "mlflow.tracking.context.default_context._get_source_type",
+        return_value=SourceType.from_string(expected_tags[mlflow_tags.MLFLOW_SOURCE_TYPE]),
+    )
+    source_version_patch = mock.patch(
+        "mlflow.tracking.context.git_context._get_source_version",
+        return_value=expected_tags[mlflow_tags.MLFLOW_GIT_COMMIT],
+    )
+
+    with multi_context(
+        source_name_patch,
+        source_type_patch,
+        source_version_patch,
+    ):
+        model = mlflow.create_logged_model()
+        assert expected_tags.items() <= model.tags.items()
```

Comment:
```suggestion
    with (
        mock.patch(
            "mlflow.tracking.context.default_context._get_source_name",
            return_value=expected_tags[mlflow_tags.MLFLOW_SOURCE_NAME],
        ) as m_get_source_name,
        mock.patch(
            "mlflow.tracking.context.default_context._get_source_type",
            return_value=SourceType.from_string(expected_tags[mlflow_tags.MLFLOW_SOURCE_TYPE]),
        ) as m_get_source_type,
        mock.patch(
            "mlflow.tracking.context.git_context._get_source_version",
            return_value=expected_tags[mlflow_tags.MLFLOW_GIT_COMMIT],
        ) as m_get_source_version,
    ):
        model = mlflow.create_logged_model()
        assert expected_tags.items() <= model.tags.items()
        m_get_source_name.assert_called_once()
        m_get_source_type.assert_called_once()
        m_get_source_version.assert_called_once()
```

- `multi_context` is not necessary in Python >= 3.9.
- `assert_called_once` to ensure the mocks are really used.